frontend/packages/ui/src/components/CookieConsent/CookieConsent.tsx

```diff
@@ -18,7 +18,7 @@ export const CookieConsent: FC<Props> = ({
   return (
     <div className={styles.wrapper}>
       <div className={styles.content}>
-        <h4 className={styles.title}>Liam Cookie Consent</h4>
+        <h4 className={styles.title}>Liam ERD Cookie Consent</h4>
```

Comment:
It looks like we need to update the VRT snapshot!🌌

https://github.com/liam-hq/liam/actions/runs/13557975017/job/37896056558?pr=788

Reply:
@FunamaYukina Umm, It looks another problem.

![top-1-diff](https://github.com/user-attachments/assets/e8246dcb-70b0-4d64-b2c6-fa2ef5d80227)

Reply:
Oh, I understand. It's a bit of a flaky test.🙏

Reply:
I have confirmed that the re-run will pass.🙏
https://github.com/liam-hq/liam/actions/runs/13557975017/job/37904199131?pr=788

Reply:
Oh, Thank you for check it!